metro/proto/v1/spec.proto

```diff
@@ -564,6 +573,10 @@ message PushConfig {
         // If specified, Pub/Sub will generate and attach an OIDC JWT token as an
         // `Authorization` header in the HTTP request for every pushed message.
         OidcToken oidc_token = 3;
+
+        // If provided, this basic auth credentials(username and password) will be used as an 
+        // `Authorization` header in the HTTP request for every pushed message
+        BasicAuth basic_auth = 4;
```

Comment:
Can you please think from outboxer handler's perspective? Will our publisher client's be able to send this as part of pubsub.Message?